templates/ironicconductor/config/01-conductor.conf

```diff
@@ -1,5 +1,8 @@
 [DEFAULT]
 # Default conductor configuration
+{{- if .GracefulShutdownTimeout }}
+graceful_shutdown_timeout={{ .GracefulShutdownTimeout }}
```

Comment:
In 2025.2 this option is moving from `[DEFAULT]` to `[conductor]` group, see https://docs.openstack.org/releasenotes/ironic/2025.2.html#relnotes-31-0-0-stable-2025-2-upgrade-notes

Lets set the value in both `[DEFAULT]` and `[conductor]` for now, and add a TODO to remove the `[DEFAULT]` entry when we move to ironic release 2025.2 or later